src/Native/Windows/clrcompression/CMakeLists.txt

```diff
@@ -74,3 +74,7 @@ GENERATE_EXPORT_HEADER( clrcompression
 )
 
 install (TARGETS clrcompression DESTINATION .)
+
+install (FILES ${CMAKE_CURRENT_BINARY_DIR}/Debug/clrcompression.pdb DESTINATION .
+    CONFIGURATIONS Debug
```

Comment:
The clrcompression cmake build is missing lots of linker and compiler args that it should have before it can be shipped. One of those is the DEBUG/Zi flag set to enable pdb generation. 

I'm working today on getting it to parity with the one we build internally. That will include generation of pdbs for both builds. I'll CC you both on the PR :)